src/Frontend.elm

```diff
@@ -986,93 +1073,146 @@ sponsorshipOption form s =
+            model.form.sponsorship
+                |> Maybe.andThen
+                    (\productId ->
+                        model.prices
+                            |> AssocList.get (Id.fromString productId)
+                            |> Maybe.map (\price -> Theme.priceAmount price.price)
+                    )
+                |> Maybe.withDefault 0
+
+        total =
+            accomTotal + grantTotal + sponsorshipTotal
+    in
+    column (Theme.contentAttributes ++ [ spacing 10 ])
+        [ text "Summary"
+        , model.form.attendees |> List.length |> (\num -> text <| "Attendance tickets x " ++ String.fromInt num ++ " – £" ++ String.fromFloat accomTotal)
+        , if List.length model.form.accommodationBookings == 0 then
+            text "No accommodation bookings"
+
+          else
+            model.form.accommodationBookings
+                |> List.group
+                |> List.map
+                    (\group -> summaryAccommodation model group)
+                |> column []
+        , Theme.viewIf (model.form.grantContribution /= "0") <|
+            text <|
+                "Opportunity grant: £"
+                    ++ model.form.grantContribution
+        , Theme.viewIf (sponsorshipTotal > 0) <|
+            text <|
+                "Sponsorship: £"
+                    ++ String.fromFloat sponsorshipTotal
+        , text <| "Total: £" ++ String.fromFloat total
+        ]
+
+
+summaryAccommodation model ( accom, items ) =
+    model.form.accommodationBookings
+        |> List.filter ((==) accom)
+        |> List.length
+        |> (\num ->
+                let
+                    total =
+                        model.prices
+                            |> AssocList.get (Id.fromString (Tickets.accomToTicket accom).productId)
+                            |> Maybe.map (\price -> Theme.priceAmount price.price)
+                            |> Maybe.withDefault 0
+                            |> (\price -> price * toFloat num)
+                in
+                Tickets.accomToString accom ++ " x " ++ String.fromInt num ++ " – £" ++ String.fromFloat total
+           )
+        |> text
+
+
+backgroundColor : Color
 backgroundColor =
-    Element.rgb255 255 244 225
-
-
-carbonOffsetForm textInput showCarbonOffsetTooltip form =
-    Element.column
-        [ Element.width Element.fill
-        , Element.spacing 24
-        , Element.paddingEach { left = 16, right = 16, top = 32, bottom = 16 }
-        , Element.Border.width 2
-        , Element.Border.color (Element.rgb255 94 176 125)
-        , Element.Border.rounded 12
-        , Element.el
+    rgb255 255 244 225
+
+
+carbonOffsetForm showCarbonOffsetTooltip form =
+    column
+        [ width fill
+        , spacing 24
+        , paddingEach { left = 16, right = 16, top = 32, bottom = 16 }
+        , Border.width 2
+        , Border.color (rgb255 94 176 125)
+        , Border.rounded 12
+        , el
             [ (if showCarbonOffsetTooltip then
```

Comment:
I've lost track a little, but there was some carbon info tooltip stuff removed earlier. Do we need it back? https://github.com/elm-camp/website/pull/34/files#diff-465a06a0ad2f1336d3ffa3242290c08c671441d35ea4564284b70377310985b1R880